connectors/sources/redis.py

```diff
@@ -270,6 +339,15 @@ async def get_docs(self, filtering=None):
         Yields:
             dictionary: Document from Redis.
         """
-        async for db in self.client.get_databases():
-            async for document in self.get_db_records(db=db):
-                yield document, None
+        if filtering and filtering.has_advanced_rules():
+            for rule in filtering.get_advanced_rules():
+                async for document in self.get_db_records(
+                    db=rule.get("database"),
+                    pattern=rule.get("key_pattern"),
+                    type_=rule.get("type_"),
```

Comment:
This is something configured by customers, I don't think it needs the suffix. 

```suggestion
                    type_=rule.get("type"),
```